subprojects/performance/src/performanceTest/groovy/org/gradle/performance/regression/java/JavaCleanAssemblePerformanceTest.groovy

```diff
@@ -17,10 +17,12 @@
 package org.gradle.performance.regression.java
 
 import org.gradle.performance.AbstractCrossVersionPerformanceTest
+import spock.lang.Ignore
 import spock.lang.Unroll
 
 import static org.gradle.performance.generator.JavaTestProject.*
 
+@Ignore('MEDIUM_JAVA_COMPOSITE_BUILD contains buildSrc')
```

Comment:
❌ Containing `buildSrc` is fine. The question is why we generate something else than our internal mirror in there? Maybe because there is no `build.gradle` in `buildSrc`? So I guess we should add that. Note that we already use the mirrors in the templates: https://github.com/gradle/gradle/blob/7e6356754f4b739b1fb635e45f09143a35dc80c8/subprojects/internal-performance-testing/src/main/groovy/org/gradle/performance/generator/JavaTestProject.groovy#L52